registry/storage/ocimanifesthandler.go

```diff
@@ -107,12 +112,13 @@ func (ms *ocischemaManifestHandler) verifyManifest(ctx context.Context, mnfst oc
 				err = distribution.ErrBlobUnknown // just coerce to unknown.
 			}
 
+			if err != nil {
```

Comment:
`err = distribution.ErrBlobUnknown` this line is useless because of `fallthrough`.

I am not sure that why check twice here. To pass the CI, I add debug log here.